tensorboard/webapp/widgets/data_table/data_table_component.ts

```diff
@@ -181,13 +178,16 @@ export class DataTableComponent implements OnDestroy, AfterContentInit {
     if (!this.draggingHeaderName || !this.highlightedColumnName) {
       return;
     }
+    const source = this.getHeaderByName(this.draggingHeaderName);
+    const destination = this.getHeaderByName(this.highlightedColumnName);
+    if (source && destination && source !== destination) {
```

Comment:
```suggestion
    const source = {...this.draggingHeaderName};
    const destination = {...this.highlightedColumnName};
    if (source && destination && source.name !== destination.name) {
```

Reply:
Doesn't work in this case because the instance variables are just names, not objects

Reply:
Ahhh, I missed that